frontend/openchat-client/src/openchat.ts

```diff
@@ -1268,8 +1271,23 @@ export class OpenChat extends OpenChatAgentWorker {
         }
     }
 
-    canCreatePolls(chatId: ChatIdentifier): boolean {
-        return this.chatPredicate(chatId, canCreatePolls);
+    canSendMessage(chatId: ChatIdentifier, inThread: boolean, permission: MessagePermission): boolean {
```

Comment:
could it be `permission?: MessagePermission` to cover the case where we don't care which specific type so that we don't actually need `canSendAnyMessages`?